elrond-wasm/src/api/helpers/managed_type_helper.rs

```diff
@@ -39,4 +39,12 @@ impl<M: ManagedTypeApi> ManagedTypeHelper<M> {
     pub fn token_identifier_egld(&self) -> TokenIdentifier<M> {
         TokenIdentifier::egld(self.api.clone())
     }
+
+    pub fn address_zero(&self) -> ManagedAddress<M> {
+        ManagedAddress::zero_address(self.api.clone())
+    }
+
+    pub fn address_const(&self, bytes: &'static [u8; 32]) -> ManagedAddress<M> {
```

Comment:
`address_from_bytes` maybe? If that's not explicit enough, `address_from_static_bytes`?